metabase/metabase
# Trend Charts — Add static number comparison

Closes #35552

Adds "static number" comparison type for trend charts

### How to verify

1. New > Question > Raw Data > Sample Database > Products
2. Summarize Count by Created At
3. Set visualization to "trend"
4. Visualization settings > Data tab > Comparisons > Custom Value
5. Try different values for "Label" and "Value" and ensure the chart calculates and displays the change as expected

### Demo

https://github.com/metabase/metabase/assets/17258145/1c41e00d-05a8-44be-8f4f-4f3a746f8914

### Checklist
